src/Helpers/IPFSClient.test.ts

```diff
@@ -0,0 +1,28 @@
+import { IPFS } from './IPFSClient';
+
+const port = 5001
+const ipfsUrl = `http://localhost:${port}`;
+
+const ipfs = new IPFS({ ipfsUrl })
+
+const run = async () => {
+  const text = 'foooooooo'
+  const addTextResult = await ipfs.addText(text);
+  console.log('addTextResult', addTextResult);
+
+  const catResult = await ipfs.cat(addTextResult.hash)
+  console.log('catResult', catResult)
+
+  const emptyDirectory = await ipfs.createEmptyDirectory()
+  console.log('emptyDirectory', emptyDirectory)
+
+  const ls = await ipfs.ls('QmcGcrtDGWvij6mmv5Uyqod1ddmycxX1MWKk5VLp8QJTkx')
+  console.log('ls', ls)
+
+  const addFileToDirectory = await ipfs.addFileToDirectory(emptyDirectory.hash, addTextResult.hash, addTextResult.hash)
+  console.log('addFileToDirectory', addFileToDirectory)
+}
+
+run()
```

Comment:
How long should it take for this to run? I seem to be stuck waiting for the 'ls' output after 2-3 minutes.

Reply:
Ipfs can take a long time to resolve a file/directory, not sure of it's time out. My ipfs node that has that directory is currently offline so it probably won't find it. You can maybe add your own directory to ls

Reply:
Can't we test `ls` with a dynamic value rather than a hard-coded one? Maybe doing the ls test against the `emptyDirectory` before and after calling `addFileToDirectory`?